test/function/samples/default-exports-of-anonymous-class/main.js

```diff
@@ -0,0 +1,3 @@
+import defaultClass from './foo.js';
+
+assert.strictEqual('default', defaultClass.name);
```

Comment:
Shouldn't the name be `undefined`?

Reply:
no, that's essentially the bug. a default exported anonymous class has the name `default`. you can try it out in node.js or in a browser.

Reply:
Indeed you are right. Funny quirk.